src/test/java/com/browserstack/SingleTest.java

```diff
@@ -5,17 +5,27 @@
 
 import org.testng.Assert;
 import org.testng.annotations.Test;
+import org.openqa.selenium.JavascriptExecutor;
 
 public class SingleTest extends BrowserStackTestNGTest {
 
     @Test
     public void test() throws Exception {
         driver.get("https://www.google.com/ncr");
+        try {
+            WebElement ele = driver.findElement(By.className("IKl7Q"));
```

Comment:
Isn't these ids are generated by the framework at runtime and can change?

Reply:
yes, I have updated the code to find the element.